spk/dnscrypt-proxy/Makefile

```diff
@@ -0,0 +1,29 @@
+SPK_NAME = dnscrypt-proxy
+SPK_VERS = 2.0.9b2
+SPK_REV = 0
+SPK_ICON = src/dnscrypt-proxy.png
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHES)
```

Comment:
Sounds inconsistent with https://github.com/SynoCommunity/spksrc/pull/3265/files#diff-141db1dbf1b0d78677131440ebf146edR38
From my point of view, as Go is available for PPC, there is no reason why application does not run on PPC. May you please investigate/confirm? And so remove `UNSUPPORTED_ARCHS` accordingly

Reply:
Go for ppc is not compiled right now: https://github.com/SynoCommunity/spksrc/blob/master/native/go/Makefile#L22 The problem is that ppc needs extra bootstrapping to compile https://dave.cheney.net/2015/10/16/bootstrapping-go-1-5-on-non-intel-platforms